app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_cp, file_name, file_to_copy = command.split()
+    if file_name == file_to_copy:
+        raise TypeError(f"{file_name} are the same name {file_to_copy}")
+    if command_cp == "cp" and file_name != file_to_copy:
+        with open("file.txt", "r") as parent_file, \
+                open("new_file.txt", "w") as file_copy:
```

Comment:
```suggestion
        with (
            open("really?????", "r") as parent_file,
            open("the same???", "w") as file_copy
        ):
```
Also file names can be not only file.txt and new_file.txt